CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ New features:
 
 Bug fixes:
 
+* Handle foreign null object as falsy value (#2055, @ssnickolay)
```

Comment:
The number here should be the issue number, not the PR number (unless there is no existing issue)